src/rest/study.ts

```diff
@@ -23,8 +31,8 @@ export function getServersInfos(token) {
             cache: 'default',
         },
         token
-    ).catch((reason) => {
-        console.error('Error while fetching the servers infos : ' + reason);
-        return reason;
-    });
+    ).catch((error) => {
+        console.error(`Error while fetching the servers infos : ${error}`);
```

Comment:
Maybe go through the getMessageError function ?

Reply:
good idea